dump/host_state_watch.cpp

```diff
@@ -43,22 +40,18 @@ void HostStateWatch::propertyChanged(sdbusplus::message::message& msg)
                 ProgressStages bootProgress =
                     sdbusplus::xyz::openbmc_project::State::Boot::server::
                         Progress::convertProgressStagesFromString(*progress);
-                if ((bootProgress == ProgressStages::SystemInitComplete) ||
-                    (bootProgress == ProgressStages::OSStart) ||
-                    (bootProgress == ProgressStages::OSRunning))
+                if (bootProgress == ProgressStages::OSRunning)
```

Comment:
One question, if the BMC Standby is enabled then the OSRunning stage won't be reached. In that case what would be the result? Are we still offloading the file? But by the looking at the code we are not apparently it seems.

Reply:
At BMC standby we do not offload the file, For NON HMC managed systems dumps are offloaded to Operating system. For operating system to start, it need to complete sbe, hostboot ipl and phyp should be loaded and then OS will be loaded. We use the partition in OS to store these dumps. This is the reason we check for OSRunning state.